{"target": "tests/test_handler.py::TestListener::test_send_group_crash", "type": "victim", "patch": false, "polluter": {"tests/transport/default/test_http.py::TestClient::test_identify": [], "tests/transport/default/test_http.py::TestClient::test_identify_retrying": [], "tests/transport/default/test_http.py::TestClient::test_logger_no_config": [], "tests/transport/default/test_http.py::TestClient::test_post_arguments": [], "tests/transport/default/test_http.py::TestClient::test_post_gzip": [], "tests/transport/default/test_http.py::TestClient::test_send_log_group": [], "tests/transport/default/test_http.py::TestClient::test_send_log_group_retrying": []}, "time": 41314.79435682297}